pkg/service/blobs/server.go

```diff
@@ -36,19 +36,7 @@ func NewBlobGetHandler(blobs blobstore.Blobstore) func(http.ResponseWriter, *htt
 	if fsblobs, ok := blobs.(blobstore.FileSystemer); ok {
 		serveHTTP := http.FileServer(fsblobs.FileSystem()).ServeHTTP
 		return func(w http.ResponseWriter, r *http.Request) {
-			_, bytes, err := multibase.Decode(r.PathValue("blob"))
```

Comment:
this might be simpler with ServeFile instead of FileServer.

also go errata: if you just want to use system utils, you may find ServeFileFS and FileServerFS easer cause you can just use some system utils (if this is actually helpful)

Reply:
Ah yes - looks more appropriate. I didn't realize it existed! If I have a moment I'll look into replacing it.